src/frontend/packages/lib_common/.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
As explain in the prettier documentation, prettier will look ahead if it founds a configuration file. We can maybe have only one file to maintain ?